src/lib.rs

```diff
@@ -48,6 +48,9 @@ pub extern crate serde;
 use alloc::borrow::Cow;
 use core::{fmt, str};
 
+#[cfg(not(feature = "std"))]
```

Comment:
```suggestion
#[cfg(all(feature = "alloc", not(feature = "std")))]
```

Otherwise we get 
```
  cargo build --no-default-features --features= --lib
   Compiling bip39 v2.1.0 (/home/tobin/build/github.com/tcharding/rust-bip39/master)
error[E0433]: failed to resolve: use of undeclared crate or module `alloc`
  --> src/lib.rs:52:5
   |
52 | use alloc::{string::ToString, vec::Vec};
   |     ^^^^^ use of undeclared crate or module `alloc`
```